src/fixtures/config.ts

```diff
@@ -17,6 +17,12 @@ const configFixture: Config = {
     sample_rate: 0.4,
     token: "d00fe9b6-91c6-4434-8e77-14630e263a26"
   },
+  client: {
+    country_code: "GB",
```

Comment:
I'm confused here. How does this type-check properly? These are supposed to be arrays, not single strings.

Reply:
Oh, I see. This is a `Client` and not a `TaskClassification`.